src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.jelly

```diff
@@ -73,6 +73,23 @@
                 </f:optionalBlock>
             </table>
         </f:block>
+        <f:block>
```

Comment:
A block in a table in a block....

I think it should suffice to only have the ` <f:optionalBlock ..>` and remove `<f:block><table>`
But it's been a while since I played with that one, so I could be wrong.